.eslintrc

```diff
@@ -56,20 +59,24 @@
     {
       "files": ["**/*.test.{ts,tsx}"],
       "rules": {
+        "jest/no-truthy-falsy": "off",
+        "shopify/jsx-no-hardcoded-content": "off",
```

Comment:
We want to check for hardcoded content to make sure we're using translated strings in our source, but we don't care about that in our tests and examples.